charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -13,9 +13,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.17.0]
```

Comment:
Hey @Simple-Analysis can you please change this `HorizontalPodAutoscaler` to 2.16.1?

Reply:
+1 This will break upgrades right?

Reply:
Sure thing!

```suggestion
## [2.16.1]
```